src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -3442,3 +3443,70 @@ def assert_Pyomo_object(b, name):
                     f"Argument {obj} of type {type(obj)} is not a supported object type. "
                     f"Ensure {name} is a Pyomo Param, Var, Expression, or ScalarExpression."
                 )
+
+    def economy_of_numbers(
+        blk, cum_num_units, cost_FOAK, CE_index_year, learning_rate=0.04
+    ):
+        """
+        Equations for Economy of Numbers (EoN) derived from NETL Quality Guidelines for Energy System Studies
+        report on Technology Learning Curve (FOAK to NOAK).
+
+        Economy of Number (EoN) estimates the future profitability of novel/First-of-A-Kind (FOAK)
+        equipment. This is because the cost of manufacturing a piece of equipment tends to reduce
+        as the cumulative production quantity rises, resulting from a consistent improvement in
+        technical know-how.
+
+        Y = A(X^-b)
+
+        b = - log(1-R)/log(2)
+        where Y is the cost of the Nth-of-A-Kind (NOAK) of the equipment, A is the cost of the FOAK,
+        X is the cumulative number of units, b is the learning rate exponent, and R is the learning
+        rate constant.
+
+        Args:
+            cum_num_units: The cumulative number of units.
+            cost_FOAK: The cost of manufacturing the First-of-A-Kind equipment.
+            CE_index_year: year for cost basis, e.g., "2021" to use 2021 dollars
+            learning_rate: ranges between 0.01 - 0.06, depending on the level of maturity
+                           (i.e., experimental, growing, proven, etc.)
+
+        """
+
+        blk.cum_num_units = Param(
+            initialize=cum_num_units,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Cumulative number of units produced",
+        )
+        blk.learning_rate = Param(
+            initialize=learning_rate,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="The learning factor reflects the level of maturity of the unit/technology",
+        )
+        blk.cost_FOAK = Param(
+            initialize=cost_FOAK,
+            mutable=True,
+            doc="Cost of the First-of-A-Kind of the unit",
+            units=getattr(pyunits, "MUSD_" + CE_index_year),
+        )
```

Comment:
Instead of making this a parameter, why not just use `cost_FOAK` directly? This will also ensure the model is dynamic, and will update/resolve if things change upstream.